src/Acheve.TestHost/Routing/TestServerArgument.cs

```diff
@@ -2,14 +2,16 @@
 {
     public class TestServerArgument
     {
-        public TestServerArgument(object instance, bool isBody = false)
+        public TestServerArgument(object instance, bool isBody = false, bool isForm = false)
```

Comment:
Rename these variable to **isFromBody** and **isFromForm**

Reply:
Fixed